pages/sf-subscription.vue

```diff
@@ -234,8 +237,9 @@ async function devDownload() {
               {{ t("pages.sfSubscription.devCheck.download") }}
             </button>
           </div>
-          <div v-if="lastUpdate" class="text-gray-500 text-sm flex gap-2">
-            {{ t("pages.sfSubscription.devCheck.lastUpdate", { time: $dayjs(lastUpdate).format("lll") }) }}
+          <div v-if="lastUpdateTime" class="text-gray-500 text-sm flex gap-2">
+            {{ t("pages.sfSubscription.devCheck.lastUpdate", {time: $dayjs(lastUpdateTime).format("lll")}) }}
+            {{ t("pages.sfSubscription.devCheck.updateInfo", {updateInfo: lastUpdateCommit}) }}
             <div v-if="noUpdate">
               {{ t("pages.sfSubscription.devCheck.noUpdate") }}
             </div>
```

Comment:
这样改会全部显示在一行里，可以考虑改成下面这样
```suggestion
          <div v-if="lastUpdateTime" class="text-gray-500 text-sm flex flex-col gap-2">
            <div class="flex gap-2">
              {{ t("pages.sfSubscription.devCheck.lastUpdate", {time: $dayjs(lastUpdateTime).format("lll")}) }}
              <div v-if="noUpdate">
                {{ t("pages.sfSubscription.devCheck.noUpdate") }}
              </div>
            </div>
            {{ t("pages.sfSubscription.devCheck.updateInfo", {updateInfo: lastUpdateCommit}) }}
```